wallet/wallet.c

```diff
@@ -2239,7 +2239,7 @@ bool wallet_network_check(struct wallet *w,
 					   "match network blockchain hash: %s "
 					   "!= %s. "
 					   "Are you on the right network? "
-					   "(--network={bitcoin, testnet, regtest, "
+					   "(--network={bitcoin, testnet, signet, regtest, "
```

Comment:
Maybe we should expose a new function in `bitcoind/chainparams.c` that returns a comma-separated string of networks we support, derived from the array of supported networks? That lets us not have to keep updating this as we add new networks or remove networks, just the chainparams table.

Reply:
I did this, but I am not sure how to trigger that function easily. See 0d8d59243b3df51564f25a0748a67f78fdd02e42